modules/openapi-generator/src/main/java/org/openapitools/codegen/IJsonSchemaValidationProperties.java

```diff
@@ -170,7 +170,7 @@ default void setTypeProperties(Schema p) {
         } else if (ModelUtils.isStringSchema(p)) {
             setIsString(true);
             if (ModelUtils.isByteArraySchema(p)) {
-                ;
+                setIsString(false);
```

Comment:
Can you please change this in Default codegen?
This layer is setting booleans according to their openpi definition.

Reply:
Done.  Could you please take a look ?